stetho-sample/src/debug/java/com/facebook/stetho/sample/APODContentProviderDumper.java

```diff
@@ -0,0 +1,112 @@
+package com.facebook.stetho.sample;
+
+import android.content.ContentResolver;
+import android.database.Cursor;
+import com.facebook.stetho.dumpapp.DumpException;
+import com.facebook.stetho.dumpapp.DumperContext;
+import com.facebook.stetho.dumpapp.DumperPlugin;
+
+import java.io.PrintStream;
+import java.util.List;
+
+public class APODContentProviderDumper implements DumperPlugin {
+
+  private static final String NAME = "content";
+
+  private static final String CMD_LIST = "list";
+  private static final String USAGE_LIST = CMD_LIST;
+
+  private static final String CMD_CLEAR = "clear";
+  private static final String USAGE_CLEAR = CMD_CLEAR;
+
+  private static final String CMD_REMOVE_ROW = "delete";
+  private static final String USAGE_REMOVE_ROW = CMD_REMOVE_ROW + " <row id>";
+
+  private final ContentResolver mContentResolver;
+
+  public APODContentProviderDumper(ContentResolver contentResolver) {
+    mContentResolver = contentResolver;
+  }
+
+  @Override
+  public String getName() {
+    return NAME;
+  }
+
+  @Override
+  public void dump(DumperContext dumpContext) throws DumpException {
+    PrintStream writer = dumpContext.getStdout();
+    List<String> rawArgs = dumpContext.getArgsAsList();
+    if (rawArgs.size() == 0) {
+      usage(writer);
+      return;
+    }
+    String command = rawArgs.get(0);
+
+    if (CMD_LIST.equalsIgnoreCase(command)) {
+      doList(writer);
+    } else if (CMD_REMOVE_ROW.equalsIgnoreCase(command)) {
+      doRemove(writer, rawArgs);
+    } else if (CMD_CLEAR.equalsIgnoreCase(command)) {
+      doClear(writer);
+    } else {
+      usage(writer);
+    }
+  }
+
+  private void doList(PrintStream writer) {
+    Cursor cursor = mContentResolver.query(
+        APODContract.CONTENT_URI,
+        new String[] {
+            APODContract.Columns._ID,
+            APODContract.Columns.TITLE,
+            APODContract.Columns.DESCRIPTION_TEXT},
+        null /* selection */,
+        null /* selectionArgs */,
+        APODContract.Columns._ID);
+
+    writer.println(APODContract.Columns._ID + "\t" +
+        APODContract.Columns.TITLE + "\t" +
+        APODContract.Columns.DESCRIPTION_TEXT);
+
+    while (cursor.moveToNext()) {
+      writer.println(String.valueOf(cursor.getInt(0)) + "\t" +
```

Comment:
Ill do long form, I think if we want to add a --tsv or --csv we can do that in a followup. The main reason is I dont want to pull in apache cli for this :)